networkx/algorithms/tests/test_planarity.py

```diff
@@ -389,40 +389,42 @@ def test_missing_edge_orientation(self):
     def test_invalid_edge_orientation(self):
         with pytest.raises(nx.NetworkXException):
             embedding = nx.PlanarEmbedding()
-            embedding.add_half_edge_first(1, 2)
-            embedding.add_half_edge_first(2, 1)
+            embedding.add_half_edge(1, 2)
+            embedding.add_half_edge(2, 1)
             embedding.add_edge(1, 3)
             embedding.check_structure()
```

Comment:
I think use of the context `pytest.raises` should build the instance **before** the context then context if for checking.  This could/should be changed elsewhere in this test file.
```python
        embedding = nx.PlanarEmbedding()
        embedding.add_half_edge(1, 2)
        embedding.add_half_edge(2, 1)
        embedding.add_edge(1, 3)
        with pytest.raises(nx.NetworkXException):
            embedding.check_structure()
```

Reply:
done